### ISSUE TITLE
Customise highlight for handlers

### ISSUE THREAD

**Issue by Isrothy** [lewis6991/satellite.nvim#23]
**Describe the solution you'd like**
Adding highlight group like `SatelliteMark` or `SatelliteDiagnosticError` for convenience of customising highlights.

**Comment by olimorris** [comment]
Just moved to this from `nvim-scrollbar` and second this as a really useful addition

**Comment by emPeeGee** [comment]
I agree with you guys: there are missing highlight options. I would like to customize the highlight color to make them more contrasted (personal preference).
